plugins/repository-hdfs/build.gradle

```diff
@@ -69,6 +84,20 @@ dependencies {
 
   hdfsFixture2 project(path: ':test:fixtures:hdfs-fixture', configuration: 'shadowedHdfs2')
   hdfsFixture3 project(path: ':test:fixtures:hdfs-fixture', configuration: 'shadow')
+
+  attributesSchema {
```

Comment:
nit: can we put that attributesSchema, attributeTypes and transform definitions on top of the dependency{} block?